tests/core/_entity/expected_data.json

```diff
@@ -0,0 +1,190 @@
\"storage_type\": \"excel\",\n\"scope\": \"SCENARIO:SCOPE\",\n\"default_path\": \"tests/data_sample/res.xlsx\",\n\"has_header\": \"True:bool\",\n\"exposed_type\": \"pandas\"\n}\n},\n\"TASK\": {\n\"default\": {\n\"function\": null,\n\"inputs\": [],\n\"outputs\": [],\n\"skippable\": \"False:bool\"\n},\n\"task_print_csv\": {\n\"function\": \"builtins.print:function\",\n\"inputs\": [\n\"dn_csv_in_1:SECTION\"\n],\n\"outputs\": [],\n\"skippable\": \"False:bool\"\n},\n\"task_print_excel\": {\n\"function\": \"builtins.print:function\",\n\"inputs\": [\n\"dn_excel_in_1:SECTION\"\n],\n\"outputs\": [],\n\"skippable\": \"False:bool\"\n},\n\"task_sum_csv\": {\n\"function\": \"tests.test_complex_application.sum:function\",\n\"inputs\": [\n\"dn_csv_in_2:SECTION\",\n\"dn_csv_in_1:SECTION\"\n],\n\"outputs\": [\n\"dn_sum_csv:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_sum_excel\": {\n\"function\": \"tests.test_complex_application.sum:function\",\n\"inputs\": [\n\"dn_excel_in_2:SECTION\",\n\"dn_excel_in_1:SECTION\"\n],\n\"outputs\": [\n\"dn_sum_excel:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_subtract_csv_excel\": {\n\"function\": \"tests.test_complex_application.subtract:function\",\n\"inputs\": [\n\"dn_sum_csv:SECTION\",\n\"dn_sum_excel:SECTION\"\n],\n\"outputs\": [\n\"dn_subtract_csv_excel:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_insert_placeholder\": {\n\"function\": \"tests.test_complex_application.return_a_number:function\",\n\"inputs\": [],\n\"outputs\": [\n\"dn_placeholder:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_mult_by_placeholder\": {\n\"function\": \"tests.test_complex_application.mult:function\",\n\"inputs\": [\n\"dn_subtract_csv_excel:SECTION\",\n\"dn_placeholder:SECTION\"\n],\n\"outputs\": [\n\"dn_mult:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_div_by_placeholder\": {\n\"function\": \"tests.test_complex_application.divide:function\",\n\"inputs\": [\n\"dn_mult:SECTION\",\n\"dn_placeholder:SECTION\"\n],\n\"outputs\": [\n\"dn_div:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_avg_div\": {\n\"function\": \"tests.test_complex_application.average:function\",\n\"inputs\": [\n\"dn_div:SECTION\"\n],\n\"outputs\": [\n\"csv_out:SECTION\"\n],\n\"skippable\": \"False:bool\"\n},\n\"task_avg_mult\": {\n\"function\": \"tests.test_complex_application.average:function\",\n\"inputs\": [\n\"dn_mult:SECTION\"\n],\n\"outputs\": [\n\"excel_out:SECTION\"\n],\n\"skippable\": \"False:bool\"\n}\n},\n\"SCENARIO\": {\n\"default\": {\n\"comparators\": {},\n\"frequency\": null,\n\"tasks\": []\n},\n\"scenario\": {\n\"comparators\": {},\n\"frequency\": null,\n\"tasks\": [\n\"task_print_csv:SECTION\",\n\"task_print_excel:SECTION\",\n\"task_sum_csv:SECTION\",\n\"task_sum_excel:SECTION\",\n\"task_subtract_csv_excel:SECTION\",\n\"task_insert_placeholder:SECTION\",\n\"task_mult_by_placeholder:SECTION\",\n\"task_div_by_placeholder:SECTION\",\n\"task_avg_div:SECTION\",\n\"task_avg_mult:SECTION\"\n]\n}\n}\n}",
```

Comment:
I believe we are missing the `core_version` attribute here

Reply:
Is this something new on 3.0? Theres no reference on the migration cli to this attribute.

Reply:
Yes, it is something new in 3.0 I believe.
We now store the `core_version` as an attribute of the Config.

This is so that when the user updates Taipy version (from 3.0.0 to 3.1.0 for example) and try to run their app that created in Taipy 3.0.0, they should receive an error saying that the entities are out of date

Reply:
I see. I have to add that to the migration scriot them, this is not covered yet.